biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery.spec.ts

```diff
@@ -4,36 +4,36 @@ describe('test for BioBattery class', () => {
   test('battery object throws an error if passed negative Energy', () => {
     const currentpower = -1;
     const maxCapacity = 5;
-    expect(() => new BioBattery(10, 5, currentpower, maxCapacity)).toThrow(
+    expect(() => new BioBattery(10, 5,  'small_battery', currentpower, maxCapacity)).toThrow(
       `Cannot create a battery with values: (${currentpower}, ${maxCapacity})`
     );
   });
 
   test('battery object throws an error if passed energy more than its capacity', () => {
     const currentpower = 10;
     const maxCapacity = 1;
-    expect(() => new BioBattery(10, 5, currentpower, maxCapacity)).toThrow(
+    expect(() => new BioBattery(10, 5,  'small_battery', currentpower, maxCapacity)).toThrow(
       `Cannot create a battery with values: (${currentpower}, ${maxCapacity})`
     );
   });
 
   test('battery returns less or equal to power requested', () => {
     const currentpower = 20;
     const maxCapacity = 30;
-    const battery = new BioBattery(10, 5, currentpower, maxCapacity);
+    const battery = new BioBattery(10, 5,  'small_battery', currentpower, maxCapacity);
     const outputenergy = 25;
     const expected = 25;
     expect(battery.supplyPower(outputenergy)).toBeLessThanOrEqual(expected);
   });
 
   test('battery is full when it is at capacity', () => {
-    const battery = new BioBattery(10, 5, 20, 20);
+    const battery = new BioBattery(10, 5,  'small_battery', 20, 20);
```

Comment:
instead of having this be a variable string, it looks like we already have a SMALL_BATTERY object. Can we add a "type" or "id" or something similar field into that object and use SMALL_BATTERY.type instead?

Reply:
I am writing the name here as is because it is not used in this case. Here it is a placeholder for the `name` requirement in the battery.  The `name` is used for labeling each gridItem on the state graph